packages/text-annotator-react/package.json

```diff
@@ -41,6 +41,7 @@
   },
   "dependencies": {
     "@neodrag/react": "^2.0.3",
-    "CETEIcean": "^1.9.2"
+    "CETEIcean": "^1.9.2",
+    "react-merge-refs": "^2.1.1"
```

Comment:
I decided that it would be more convenient to use a small and maintained helper package than to add the utility manually 🤷🏻‍♂️